From 89a64246a6cf832442d0a028d6f9fe43d196e24d Mon Sep 17 00:00:00 2001
From: Marina Calvo <mcalvo@rbz.es>
Date: Fri, 17 Dec 2021 12:44:06 +0000
Subject: [PATCH] Add MOD_IMX8M_PLUS support

Signed-off-by: Marina Calvo <mcalvo@rbz.es>
---
 iMX8M/soc.mak | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index 4bfdc37..d043454 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -147,9 +147,9 @@ u-boot.itb: $(dtb)
 	./$(PAD_IMAGE) tee.bin
 	./$(PAD_IMAGE) bl31.bin
 	./$(PAD_IMAGE) u-boot-nodtb.bin $(dtb)
-	DEK_BLOB_LOAD_ADDR=$(DEK_BLOB_LOAD_ADDR) TEE_LOAD_ADDR=$(TEE_LOAD_ADDR) ATF_LOAD_ADDR=$(ATF_LOAD_ADDR) ./mkimage_fit_atf.sh $(dtb) > u-boot.its
+	DEK_BLOB_LOAD_ADDR=$(DEK_BLOB_LOAD_ADDR) TEE_LOAD_ADDR=$(TEE_LOAD_ADDR) ATF_LOAD_ADDR=$(ATF_LOAD_ADDR) ./mkimage_fit_atf.sh mod-imx8m-plus.dtb > u-boot.its
 	./mkimage_uboot -E -p 0x3000 -f u-boot.its u-boot.itb
-	@rm -f u-boot.its $(dtb)
+	@rm -f u-boot.its
 
 dtb_ddr3l = valddr3l.dtb
 $(dtb_ddr3l):
-- 
2.17.1

